logexporter/cmd/main.go

```diff
@@ -51,7 +51,7 @@ var (
 
 	// Node-type specific logfiles.
 	// Currently we only handle nodes, and neglect master.
-	nodeLogs = []string{"kube-proxy", "node-problem-detector", "fluentd"}
+	nodeLogs = []string{"kube-proxy", "node-problem-detector", "fluentd", "dns-lookup", "ping-client", "ping-server"}
```

Comment:
I wonder whether we could do it more generically. E.g. collecting everything that starts with cl2-probe? This way we wouldn't have to update this file every time we add a new probe in CL2. WDYT?